src/main/resources/swagger-ui/yaml/settings.yaml

```diff
@@ -13,6 +13,25 @@ tags:
   - name: settings
     description: Operations about settings
 paths:
+  /api/v1/settings/port:
+    get:
+      summary: Artipie port.
```

Comment:
I'd suggest to specify what exactly port is it: `Artipie server-side port (repositories default port)`

Reply:
Thanks, added